### PR TITLE
FEATURE: package definition support in edb configuration

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

File: src/pyedb/dotnet/edb_core/configuration.py [hunk 1]
```diff
@@ -678,3 +682,33 @@ def _load_padstacks(self):
...
+
+    @pyedb_function_handler
+    def _load_package_def(self):
+        """Imports package definition information from JSON."""
+        comps = self._pedb.components.components
+        for pkgd in self.data["package_definitions"]:
+            name = pkgd["name"]
+            if name in self._pedb.definitions.package:
+                self._pedb.definitions.package[name].delete()
+            from pyedb.dotnet.edb_core.definition.package_def import PackageDef
```
Comment (SMoraisAnsys) [inline src/pyedb/dotnet/edb_core/configuration.py:694]: Is it possible to move that call at the top of the module or does it creates circular import ?

**Review by SMoraisAnsys — APPROVED** [review]
Just a minor comment, otherwise LGTM

**Comment by SMoraisAnsys** [comment]
@ring630 Why did you added dotnet label ?

**Review by svandenb-dev — DISMISSED** [review]
LGTM

File: src/pyedb/dotnet/edb_core/configuration.py (hunk 1 above)
Comment (ring630) [reply src/pyedb/dotnet/edb_core/configuration.py:694]: No. I will correct this.

**Review by svandenb-dev — APPROVED** [review]
LGTM

**Pull request merged**